controllers/comments.controller.js

```diff
@@ -28,4 +30,19 @@ function postNewComment(request, response, next) {
     })
     .catch(next);
 }
-module.exports = { getCommentsByArticleId, postNewComment };
+
+function removeCommentByCommentId(request, response, next) {
+  const { comment_id } = request.params;
+
+  // check the comment exists first
+  return selectCommentByCommentId(comment_id)
+    .then(() => deleteCommentByCommentId(comment_id))
+    .then(() => response.status(204).send())
+    .catch(next);
+}
```

Comment:
You can simplify your code by removing the extra step of checking if the comment exists before deleting it. The database will handle this for you 😄

Reply:
Resolved in https://github.com/dku7/be-nc-news/pull/19